packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -38,7 +37,9 @@ class GoogleMapController {
   ///
   /// Accessible only for testing.
   @visibleForTesting
-  final MethodChannel channel;
+  MethodChannel get channel {
+    return _googleMapsFlutterPlatform.channel;
+  }
```

Comment:
The plugin cannot use the MethodChannel for anything. The MethodChannel is only used as an implementation of the GoogleMaps platform for certain devices. We need to remove the need for an instance of the channel within the plugin (I'll give this a try later as well!)

Reply:
This is required for the e2e tests (which won't work in mobile, then). This is an issue for the full migration.